scripts/generate/run.js

```diff
@@ -1,7 +1,30 @@
 #!/usr/bin/env node
 
 const fs = require('fs');
+const format = require('./format-lines');
 
-// SafeCast.sol
-fs.writeFileSync('./contracts/utils/math/SafeCast.sol', require('./templates/SafeCast'));
-fs.writeFileSync('./contracts/mocks/SafeCastMock.sol', require('./templates/SafeCastMock'));
+function getVersion (path) {
+  try {
+    return fs
+      .readFileSync(path)
+      .toString()
```

Comment:
The second argument specifies the encoding to get back a string, otherwise it will return a Buffer:

```suggestion
      .readFileSync(path, 'utf8')
```